lib/components/forms/NewTeamForm.js

```diff
@@ -54,7 +54,6 @@ class NewTeamForm extends React.Component {
           const teamExists = Object.keys(checkTeam).length !== 0
           if (!teamExists) {
             const team = await apiRequest(null, 'put', `/teams/${canonicalTeamName}`, Team(canonicalTeamName, spec))
-            await apiRequest(null, 'put', `/teams/${canonicalTeamName}/members/${this.props.user.id}`)
```

Comment:
assume this is another one done by the server now?

Reply:
Yes indeed, the user who creates the team is added as a member by Kore. It didn't used to be like that.